File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_quantile.R
library(h2o)
h2o.init(nthreads = -1)
train.hex <- h2o.importFile("https://h2o-public-test-data.s3.amazonaws.com/smalldata/iris/iris_wheader.csv")
splits <- h2o.splitFrame(train.hex, 0.75, seed=1234)
dl <- h2o.deeplearning(x=1:3, y="petal_len",
        training_frame=splits[[1]],
        distribution="quantile", quantile_alpha=0.8)
h2o.predict(dl, splits[[2]])

File: h2o-docs/src/booklets/source/DeepLearning_Vignette_code_examples/deeplearning_quantile.py
import h2o
from h2o.estimators.deeplearning import H2ODeepLearningEstimator
h2o.init() 
train = h2o.import_file("https://h2o-public-test-data.s3.amazonaws.com/smalldata/iris/iris_wheader.csv")
splits = train.split_frame(ratios=[0.75], seed=1234)
dl = H2ODeepLearningEstimator(distribution="quantile", quantile_alpha=0.8)
dl.train(x=range(0,2), y="petal_len", training_frame=splits[0])
print(dl.predict(splits[1]))
